examples/inspec-linux-audit/habitat/default.toml

```diff
@@ -45,7 +45,7 @@ log_level = "warn"
 #  - user: User for requests to Chef Automate (Default: '<automate_user>')
 [automate]
 enable = false
-server_url = "https://<automate_url>/data-collector/v0/"
+server_url = "https://<automate_url>"
```

Comment:
This is a problem that is currently not solved I believe. We should keep it as is, until we solve it.

Reply:
This is solved by this PR the `/data-collector/v0` is now set in the `inspec_exec_config.json`

Reply:
Ohhhhhhh, I see now. Sorry, it's just inconsistent between infra and inspec. It totally works here,  you're right!